editor/osm_auth.hpp

```diff
@@ -0,0 +1,122 @@
+#pragma once
+
+#include "std/string.hpp"
+#include "std/utility.hpp"
+
+namespace osm
+{
+
+using TKeySecret = pair<string /*key*/, string /*secret*/>;
```

Comment:
Please, do not use using declarations at namespace level in headers. Move them to the corresponding class scope.